test/support/page_objects/settings_page.js

```diff
@@ -31,7 +31,7 @@ export default class SettingsPage {
   }
 
   clickExistingData() {
```

Comment:
I could just remove it and switch all usages to `clickKibanaIndices`, how's that?